docs/environments/atari/gopher.md

```diff
@@ -52,8 +52,7 @@ See variants section for the type of observation used by each environment id by
 The exact reward dynamics depend on the environment and are usually documented in the game's manual. You can
 find these manuals on [AtariAge](https://atariage.com/manual_html_page.php?SoftwareLabelID=218).
 
-Atari environments are simulated via the Arcade Learning Environment (ALE) [[1]](#1).
-
+Atari environments are simulated via the Arcade Learning Environment (ALE) [[1]](
```

Comment:
This is strange, do you know why this is happening?

Reply:
whoops bad grep on my part- fixed